packages/api/src/domain/medical/ihe-result.ts

```diff
@@ -1,44 +1,12 @@
 import { BaseDomainCreate } from "../base-domain";
+import { OperationOutcome } from "@metriport/ihe-gateway-sdk";
```

Comment:
We should probably leave these types here since they are specifically used by the domain. Unless there is some shared logic you are using the OperationOutcome for? We dont wont the app relying on the package for type definitions afaik